### DISCUSSION TITLE
Use traditional Rmd output formats with Quarto

### DISCUSSION THREAD

**Discussion by jtlandis** [quarto-dev/quarto-cli#7450] [Q&A]
### Description

R has a few packages that extend RMD such that the yaml header can specify something like

````rmd
---
output:
    posterdown::posterdown_html
---
````

This allows you to knit the document a certain way.

Since quarto still converts to an md document, is it possible to still leverages this packages with quarto's render function?

**Comment by mcanouil** [comment]
- **No, you have to make those formats as extension formats, see <https://quarto.org/docs/extensions/listing-formats.html>.**
- *Yes, make Quarto outputs markdown and use markdown within rmarkdown, but any Quarto specific features such as callouts, tabsets, etc. will be lost.*

Related discussion:
- https://github.com/quarto-dev/quarto-cli/discussions/2205

Also with Quarto Dashboard (1.4 pre-release) you could get a "poster".

**Comment by cderv** [comment]
To give you more details on the why: 

`posterdown_html()` is a R Markdown output format. Quarto does not know how to handle it. So you `quarto render` will do nothing with this information, and default to `format: html` to render from the .Rmd source.  

This formats defines the pre/post processor for **knitr** and **pandoc**, along with pandoc options so that `rmarkdown::render()` knows what to do. 

Output formats like `posterdown_html()` is also a R function. And Quarto does only use R for running **knitr** to evaluate document content. Any pre/post processing is not done in R, including calling Pandoc for the convertion.  

So as you can see this is quite **rmarkdown** specific. 

> Since quarto still converts to an md document, is it possible to still leverages this packages with quarto's render function?

You're right to think that Quarto does the same logic to convert to .md than calling Pandoc. However, it does it completely differently, and it requires a Quarto format to define information differently. That is what Quarto extensions are for. 

`posterdown_html()` will produce a output file though. If you are working with Quarto project and wants to include the results of your R Markdown document rendered with `rmarkdown::render()`, you could do so though, by including it as external resources. You would need[ to ignore the file to be rendered by Quarto](https://quarto.org/docs/projects/quarto-projects.html#render-targets), and include the results as a resource for your project (https://quarto.org/docs/websites/website-tools.html#site-resources) to be included in the output. Though, the style and template would be different - it is really external. 

Anyhow, I hope this clarifies. 

To make poster with Quarto, you could have a look at Typst too (as part of next 1.4):  
* https://quarto.org/docs/prerelease/1.4/typst.html
* https://github.com/quarto-ext/typst-templates/tree/main/poster

Or follow this thread for discussion: 
* https://github.com/quarto-dev/quarto-cli/discussions/2205

Hope it helps
